app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -1,40 +1,27 @@
-import React, { useState } from "react";
+import React from "react";
 
 import { Alert } from "neetoui";
 
-import notesApi from "apis/notes";
-
 const DeleteAlert = ({
-  refetch,
   onClose,
-  selectedNoteIds,
-  setSelectedNoteIds,
+  selectedNoteId,
+  setNotes,
+  setShowDeleteAlert,
 }) => {
-  const [deleting, setDeleting] = useState(false);
-
-  const handleDelete = async () => {
-    try {
-      setDeleting(true);
-      await notesApi.destroy({ ids: selectedNoteIds });
-      onClose();
-      setSelectedNoteIds([]);
-      refetch();
-    } catch (error) {
-      logger.error(error);
-      setDeleting(false);
-    }
+  const deleteNote = noteId => {
```

Comment:
We should follow the pattern of noun-verb format for naming functions that handle something.

Here this function handles the deletion of a note. Thus the better name would be `handleNoteDeletion`. But given that we are already within the `Notes` namespace, we can drop the note keyword, and keep the name as `handleDelete`.

Refer: https://stackoverflow.com/a/60048346